src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1419,7 +1419,7 @@
   <data name="DateModifiedLowerCase" xml:space="preserve">
     <value>Date modified</value>
   </data>
-  <data name="NavToolbarArrangementOptionOriginalFolder.Text" xml:space="preserve">
```

Comment:
Can you search to see if this is still being used? Same for the other strings that were removed.

Reply:
I had already checked and I just checked again.